server/rock-ci-metadata.yaml

```diff
@@ -0,0 +1,5 @@
+integrations:
+- consumer-repository: https://github.com/canonical/kubeflow-model-registry-operators.git
+  replace-image:
+    - file: charms/server/metadata.yaml
```

Comment:
suggestion: the charm directory follows the charm name, we should name it something specific to model registry e.g. `model-registry-server`.